modules/escalaxBidAdapter.js

```diff
@@ -0,0 +1,84 @@
+import { ortbConverter } from '../libraries/ortbConverter/converter.js';
+import { registerBidder } from '../src/adapters/bidderFactory.js';
+import { config } from '../src/config.js';
+import { BANNER, NATIVE, VIDEO } from '../src/mediaTypes.js';
+
+const BIDDER_CODE = 'escalax';
+const ESCALAX_SOURCE_ID_MACRO = '[sourceId]';
+const ESCALAX_ACCOUNT_ID_MACRO = '[accountId]';
+const ESCALAX_SUBDOMAIN_MACRO = '[subdomain]';
```

Comment:
What is the goal of the dynamic subdomain?

Reply:
@patmmccann
We have a different subdomain for each region - us, eu and so on